src/boot/efi/boot.c

```diff
@@ -1666,15 +1667,9 @@ static void config_load_type1_entries(
 }
 
 static int boot_entry_compare_indirect(const BootEntry **a_ind, const BootEntry **b_ind) {
-        const BootEntry *a, *b;
+        const BootEntry *a = *ASSERT_PTR(a_ind), *b = *ASSERT_PTR(b_ind);
         int r;
 
-        assert(a_ind);
-        assert(b_ind);
-
-        a = *a_ind;
-        b = *b_ind;
```

Comment:
this is a fixup of an earlier commit in the same PR. please just make the right change right-away